### ISSUE TITLE
Error: Could not locate KeePass Password Safe 2.x installation location.

### ISSUE THREAD

**Issue by Yellephant** [pfn/keepasshttp#321]
I get the title Error message when I try to install keepasshttp with chocolatey.

When I first tried to install keepasshttp I had KeepAss Password Safe 2.x as well as keepassx 2.x installed. I only use the latter, but the former is where keepasshttp got installed. I probably didn't handle it in the best way, but my current situation is that KeepAss Password Safe 2.x is uninstalled and keepassx 2.x is still being used. I need to help keepasshttp to find keepassx 2.x. How do I do it? Thank you.

**Comment by kendaleiv** [comment]
The [`keepass-keepasshttp`](https://chocolatey.org/packages/keepass-keepasshttp) Chocolatey package has a dependency on the [`keepass`](https://chocolatey.org/packages/keepass) Chocolatey package, which is probably why it is being installed. For KeePassX I recommend a manual installation at this time.

**Comment by Yellephant** [comment]
Thanks for the quick answer.

This is what I did:

From the site https://github.com/pfn/keepasshttp : Under the title Non-Windows / Manual Windows installation: clicked on the link in step 1.

Per these instructions https://github.com/pfn/passifox/issues/261 :

- Put the downloaded file named KeePassHttp.plgx into the Keepassx folder, in the same folder as KeePassx.exe
- Restarted keepasssx
- open the chromeIPass popup - but there is no Connect button. Instead it's a Reload button.
- Clicked Settings, see this: Your running version of KeePassHttp: unknown
- Clicked Connected Databases tab. See this: No connected databases found.

I will try switching to keepass 2.x until I have found a solution.
